src/containers/Home/Keys.tsx

```diff
@@ -178,13 +183,14 @@ const Keys = (props) => {
                                             apiKeys[selectedKey].environment
                                         )} Node`}
                                     </Typography>
-                                    {mainUrls[apiKeys[selectedKey].environment].tezos.map( url =>
-                                    <Grid container alignItems="center">                                        
-                                        <LinkText>
+                                    {mainUrls[apiKeys[selectedKey].environment].tezos.map( (url, index) =>
+                                    <Grid container alignItems="center" key={`parent_${index}`}>                                        
+                                        <LinkText key={`first_child_${index}`}>
```

Comment:
is there any case that children need a key?

Reply:
If we remove keys in LinkText, we have some warnings in console log.